src/mds/Server.cc

```diff
@@ -5114,6 +5114,8 @@ void Server::handle_client_setattr(MDRequestRef& mdr)
       pi.inode->truncate(old_size, req->head.args.setattr.size);
       le->metablob.add_truncate_start(cur->ino());
     } else {
+      if (mask & CEPH_SETATTR_FSCRYPT_FILE)
+	pi.inode->truncate_seq++;
```

Comment:
@jtlayton 

Let's abandon this commit as discussed in the kclient mail list. It's possibly will increase the `truncate_seq` when truncating a bigger size, which will introduce data losing bug in client side.

My size handling feature patch could correctly handle this case, this commit makes no sense any more.

Thanks.

Reply:
Sounds good. Let's close this PR out and we'll just work with yours going forward.

Reply:
Sure.